data/ecosystems/a/astar-network.toml

```diff
@@ -528,9 +562,24 @@ url = "https://github.com/Phala-Network/subbridge.js"
 [[repo]]
 url = "https://github.com/PlasmNetwork/defillama-app"
 
+[[repo]]
+url = "https://github.com/rairtech/solidity"
```

Comment:
how's this repo using Astar? eg, can you point out where in the code I can find traces of Astar usage?